components/compositing/compositor.rs

```diff
@@ -1957,17 +1957,15 @@ impl<Window: WindowMethods> IOCompositor<Window> {
         let clip_rect_for_children = if masks_to_bounds {
             Rect::new(Point2D::zero(), clipped_layer_bounds.size)
```

Comment:
masks_to_bounds is only set when it's the root layer for an iframe, I think - but @mrobinson  knows this stuff much better then me.